pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -168,6 +169,18 @@ type PagerDutyConfigDetail struct {
 	Value string `json:"value"`
 }
 
+type WeChatConfig struct {
```

Comment:
There are a couple of extra properties in the upstream type that are not reflected here:

* [HTTPConfig](https://github.com/prometheus/alertmanager/blob/master/config/notifiers.go#L397)
* [MessageType](https://github.com/prometheus/alertmanager/blob/master/config/notifiers.go#L407)

Do you plan to add those?

Reply:
my bad, I only checked the alertmanager doc website, not finding those two configs, added!